cli/command/secret/create.go

```diff
@@ -46,10 +46,6 @@ func runSecretCreate(dockerCli *command.DockerCli, options createOptions) error
 	client := dockerCli.Client()
 	ctx := context.Background()
 
-	if options.file == "" {
-		return fmt.Errorf("Please specify either a file name or STDIN ('-') with --file")
```

Comment:
Should we keep this check in place? e.g.;

```bash
$ docker secret create foo ""
open : no such file or directory
```

Printing this would be slightly more informative, I think;

```
Please specify either a file name or STDIN ('-')
```